consumer/consumer.go

```diff
@@ -420,8 +401,12 @@ func (c *Consumer) consumeState(ch AMQPChannel, queue string, connCloseCh <-chan
 	defer workerCancelFunc()
 
 	c.logger.Printf("[DEBUG] consumer ready")
+	state := State{
+		Unready: false,
+		Ready:   &Ready{Queue: queue},
+	}
 
-	c.notifyReady(queue)
```

Comment:
revert notifyReady(queue) and notifyUnready(err)

use them instead of setState